docs/reference/ml/anomaly-detection/apis/get-snapshot.asciidoc

```diff
@@ -58,52 +63,136 @@ include::{docdir}/ml/ml-shared.asciidoc[tag=job-id-anomaly-detection]
 [[ml-get-snapshot-results]]
 ==== {api-response-body-title}
 
-The API returns the following information:
+The API returns an array of model snapshot objects, which have the following 
+properties:
 
-`model_snapshots`::
-  (array) An array of model snapshot objects. For more information, see
-  <<ml-snapshot-resource>>.
+`description`::
+(string) An optional description of the job.
+
+`job_id`::
+(string) A numerical character string that uniquely identifies the job that
+  the snapshot was created for.
+  
+`latest_record_time_stamp`::
+(date) The timestamp of the latest processed record.
+
+`latest_result_time_stamp`::
+(date) The timestamp of the latest bucket result.
+
+`min_version`::
+(string) The minimum version required to be able to restore the model snapshot.
+
+`model_size_stats`::
+(object) Summary information describing the model.
+
+`model_size_stats`.`bucket_allocation_failures_count`:::
+(long) The number of buckets for which entities were not processed due to memory
+limit constraints.
+
+`model_size_stats`.`job_id`:::
+(string)
+include::{docdir}/ml/ml-shared.asciidoc[tag=job-id-anomaly-detection]
+
+`model_size_stats`.`log_time`:::
+(date) The timestamp that the `model_size_stats` were recorded, according to
+server-time.
+
+`model_size_stats`.`memory_status`:::
+(string) The status of the memory in relation to its `model_memory_limit`.
+Contains one of the following values.
++
+--
+* `hard_limit`: The internal models require more space that the configured
+memory limit. Some incoming data could not be processed.
+* `ok`: The internal models stayed below the configured value.
+* `soft_limit`: The internal models require more than 60% of the configured
+memory limit and more aggressive pruning will be performed in order to try to
+reclaim space.
+--
+
+`model_size_stats`.`model_bytes`:::
+(long) An approximation of the memory resources required for this analysis.
+
+`model_size_stats`.`model_bytes_exceeded`:::
+TBD
```

Comment:
```suggestion
(long) The number of bytes over the high limit for memory usage at the last allocation failure.
```

Reply:
@edsavage I assume these definitions you provided in https://github.com/elastic/elasticsearch/pull/50057 work here too?